paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/PaymentDetails.kt

```diff
@@ -30,60 +28,66 @@ import androidx.compose.ui.Alignment
 import com.stripe.android.link.theme.linkShapes
+import com.stripe.android.link.ui.ErrorText
+import com.stripe.android.link.ui.ErrorTextStyle
 import com.stripe.android.model.CardBrand
 import com.stripe.android.model.ConsumerPaymentDetails
 import com.stripe.android.model.ConsumerPaymentDetails.Card
+import com.stripe.android.model.CvcCheck
 import com.stripe.android.paymentsheet.R
 import com.stripe.android.paymentsheet.paymentdatacollection.ach.transformBankIconCodeToBankIcon
 import com.stripe.android.paymentsheet.ui.getCardBrandIconForVerticalMode
 import com.stripe.android.R as StripeR
 
 @Composable
+@Suppress("LongMethod")
 internal fun PaymentDetailsListItem(
     modifier: Modifier = Modifier,
     paymentDetails: ConsumerPaymentDetails.PaymentDetails,
+    isAvailable: Boolean,
     enabled: Boolean,
     isSelected: Boolean,
     isUpdating: Boolean,
     onClick: () -> Unit,
     onMenuButtonClick: () -> Unit
 ) {
-    Row(
+    // Using a `Layout` in order to achieve the following:
+    //  1. main content (radio button, description, menu/loader) centered vertically
+    //  2. error below the main content
+    //  3. error start-aligned with description
+    // without knowing the dimensions of the radio button on the first layout pass.
+    Layout(
         modifier = modifier
-            .fillMaxWidth()
-            .defaultMinSize(minHeight = 56.dp)
+            .padding(top = 8.dp, bottom = 8.dp, start = 20.dp)
             .clickable(enabled = enabled, onClick = onClick),
-        verticalAlignment = Alignment.CenterVertically
-    ) {
-        RadioButton(
-            selected = isSelected,
-            onClick = null,
-            modifier = Modifier
-                .testTag(WALLET_PAYMENT_DETAIL_ITEM_RADIO_BUTTON)
-                .padding(start = 20.dp, end = 12.dp),
-            colors = RadioButtonDefaults.colors(
-                selectedColor = MaterialTheme.linkColors.actionLabelLight,
-                unselectedColor = MaterialTheme.linkColors.disabledText
+        content = {
+            RadioButton(
+                selected = isSelected,
+                onClick = null,
+                modifier = Modifier
+                    .testTag(WALLET_PAYMENT_DETAIL_ITEM_RADIO_BUTTON)
+                    .padding(end = 12.dp),
+                colors = RadioButtonDefaults.colors(
+                    selectedColor = MaterialTheme.linkColors.actionLabelLight,
+                    unselectedColor = MaterialTheme.linkColors.disabledText
+                )
             )
-        )
 
-        Column(
-            modifier = Modifier
-                .padding(vertical = 16.dp)
-                .weight(1f)
-        ) {
             Row(
-                modifier = Modifier.fillMaxWidth(),
+                modifier = Modifier.padding(vertical = 8.dp),
                 verticalAlignment = Alignment.CenterVertically
             ) {
                 PaymentDetails(paymentDetails = paymentDetails)
```

Comment:
On iOS, we show the unavailable payment methods with a lower opacity (see [here](https://github.com/stripe/stripe-ios/blob/be25b5037f54e40c913e7d9aa6cc846578da9b06/StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Components/PaymentMethodPicker/LinkPaymentMethodPicker-Cell.swift#L212)). Can you also add something like that?